custom_components/econnect_metronet/devices.py

```diff
@@ -175,6 +175,20 @@ def update(self):
         # Update the internal state machine (mapping state)
         self.state = self.get_state()
 
+    @property
```

Comment:
I would suggest to move these property up in the file. Properties are behaving like attributes (but they are not attributes), so for readability it's good if they are located right after the `__init__()` (constructor).